packages/repack/src/commands/bundle.ts

```diff
@@ -19,51 +20,59 @@ import { getWebpackConfigPath } from './utils/getWebpackConfigPath';
  * @category CLI command
  */
 export async function bundle(
-  _: string[],
-  config: Config,
-  args: BundleArguments
-) {
-  const webpackConfigPath = getWebpackConfigPath(
-    config.root,
-    args.webpackConfig
-  );
-  const cliOptions = {
-    config: {
-      root: config.root,
-      reactNativePath: config.reactNativePath,
-      webpackConfigPath,
-    },
-    command: 'bundle',
-    arguments: {
-      bundle: args,
-    },
-  } as CliOptions;
+	_: string[],
+	config: Config,
+	args: BundleArguments
+){
```

Comment:
Switching to tabs make it hard to see your actual changes. Also, since the rest of the project seems to be using two spaces for indents, I think switching to tabs is perhaps not a good idea